mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
+	name       string
+	snapshots  []*rds.DBSnapshot
+	instanceID string
+}
+
+var listSnapshotsCases = []listSnapshotsCase{
```

Comment:
Ah okay, so `{ ... }` is defining an array when they type is prefixed with `[]`?